om-2.0.rdf

```diff
@@ -22126,6 +22207,8 @@
     <om:commonlyHasUnit rdf:resource="&om;gigavolt"/>
     <om:commonlyHasUnit rdf:resource="&om;hectovolt"/>
     <om:commonlyHasUnit rdf:resource="&om;kilovolt"/>
+    <om:commonlyHasUnit rdf:resource="&om;kilovoltampere"/>
+    <om:commonlyHasUnit rdf:resource="&om;kilovoltamperereactive"/>

```

Comment:
Incorrect as explained above. Please remove.

Reply:
Removed and corrected.